### ISSUE TITLE
Make customized metadata work for TypeScript + Angular4 + WebPack2

### ISSUE THREAD

**Issue by srichandradeep** [halt-hammerzeit/libphonenumber-js#86]
Hi,

I'm using your library for my application which is built using TS+Angular4+WebPack2. I could use the library with default metadata.min.json without any issues. But, I see some issues while using a customized metadata of the library.

Details:
1. I generated the metadata.min.json file using "libphonenumber-metadata": "libphonenumber-generate-metadata metadata.min.json --types mobile" and "npm run libphonenumber-metadata".
2. I created a new TS file, metadata.ts. I have attached the file (as a .txt version).
3. When I try to use asYouType like, this.inputPhoneNumber = new asYouTypeCustom('US').input(this.inputPhoneNumber);
TypeScript compiler is throwing the error, property 'input' doesn't exist on type 'asYouType'.
4. To avoid the error, I modified the line to, this.inputPhoneNumber = new asYouTypeCustom('US').input(this.inputPhoneNumber);
It resolved the issue mentioned in the above step. But, I know that is not what is expected. 
5. Now, I see console errors saying metadata is not passed to asYouType(). This is happening is because the metadata that got imported from 'metadata.json' doesn't have any value and it is undefined.

To sum it up, I have two issues. One is 'input' isn't available on asYouType and 'metadata' variable doesn't have any data after importing from the JSON.

I'm not sure whether I'm using the customized version properly or the code doesn't support TypeScript.
Can you please help me out to resolve my issue?

Attached two files, metadata.txt and phone-number.txt.
[metadata.txt](https://github.com/halt-hammerzeit/libphonenumber-js/files/1113439/metadata.txt)
[phone-number.txt](https://github.com/halt-hammerzeit/libphonenumber-js/files/1113440/phone-number.txt)

Thank you in advance.

**Comment by halt-hammerzeit** [comment]
> One is 'input' isn't available on asYouType

I'm not familiar with TypeScript so you're on your own here.
There are no guarantees any `.ts` files are up-to-date.
If you think they aren't then you can create a Pull Request.

> and 'metadata' variable doesn't have any data after importing from the JSON

This sentence doesn't make sense.
A `.json` file, when imported, always has some data.

**Comment by vpillinger** [comment]
declare module 'libphonenumber-js' {
    const libphonenumber: any;
    export = libphonenumber;
}
What I did is add something like the above to my typings.d.ts file and called it a day on auto-completion.  Typescript just handles importing normal JavaScript libraries like complete crap.
